docs/SUMMARY.md

```diff
@@ -1,66 +1,72 @@
-<!-- markdownlint-disable -->
-# Summary
+# Quilt Documentation
 
-* [Introduction](README.md)
-* [Installation](Installation.md)
-* [Quick start](Quickstart.md)
-* [Mental model](MentalModel.md)
-* [Architecture](Architecture.md)
+* [About Quilt](README.md)
+* [Architecture](Architecture.md)  
+* [Mental Model](MentalModel.md)
+* [Metadata Management](Catalog/Metadata.md)  
+* [Metadata Workflows](advanced-features/workflows.md)
 
-### Walkthrough
-* [Editing a Package](walkthrough/editing-a-package.md)
-* [Uploading a Package](walkthrough/uploading-a-package.md)
-* [Installing a Package](walkthrough/installing-a-package.md)
-* [Getting Data from a Package](walkthrough/getting-data-from-a-package.md)
-* [Working with the Catalog](walkthrough/working-with-the-catalog.md)
-* [Working with a Bucket](walkthrough/working-with-a-bucket.md)
-* [Working with Elasticsearch](walkthrough/working-with-elasticsearch.md)
+## Quilt Platform Usage (Catalog UI)
```

Comment:
Doesn't "Catalog" and not "(Catalog UI)" deserve to be a top-level section under platform? Where do we explain the components of the platform. How do visitors know the API is free forever and the catalog is open source but offers a paid product?

Reply:
So, I could use help here.  Marketing-wise, we talk about the 'Quilt Platform'.  But for a user, all they see (and call it) is the "Quilt Catalog."  Which technically is open source, but for 99% of our users is actually the thing they think they are paying for.

My concern is that if we just call it "Platform" users won't know where to look, which is why I added the parenthetical.

Do we want to refactor our documentation so there is a hard split between the open source Catalog, and those features that are only part of the paid product? 

Or should we focus on the "Platform" vs "SDK" split, and let the README explain the nuances?

Reply:
Perhaps call this section: "Quilt Platform User Catalog"?

Reply:
@QuiltSimon What do you think of renaming each section by the people who should use it:

## Quilt Platform Catalog User
## Quilt Platform Administrator
## Quilt Python SDK Developers

A little wordy, but it seems like it would make it easier for people to scan for what they need.
Though that does make this one a bit awkward:
## Quilt Ecosystem Integrations

Reply:
I'm trying to understand the nuances of this discussion. I'm going to bring us back to first principles to get us back on track.

I think of Quilt as being our entire product, Consisting of a Platform (the catalog) an SDK that interacts with the platform (python and CLI) and then ecosystem products that extend Quilt. Id like to see the breakdown as follows

Intro
Quilt Platform
- Users
- Admin
Quilt SDK
- Python
- CLI
Platform Ecosystem
- QuiltSync
- Benchling Packager
- Nextflow Integration (nf-quilt)
- Technology Previews

I believe we all can get aligned on the fact that 

Platform == Catalog
SDK == Python and or CLI

I can be swayed on replacing the word "Platform" from "Platform Ecosystem" to make this more logically consistent if others can suggest a better replacement. Perhaps "Quilt Ecosystem".

Does this help put this issue on firmer ground? I'd be happy to pick up this PR and rearrange the docs to fit the schema I suggest here.

Reply:
I agree with the 'Concept' that Platform = Catalog.  The challenges I am concerned about are:

1.  Users think they are using the Catalog. When they come here, how do they know they should be looking under Platform?
2. Are you using "Platform" as synonymous with the paid product?  We (for better and worse) have an open source Catalog.  We don't need to talk about it much, but we do need to be able to refer to it without causing extra confusion.

I'm fine with you revising (or forking) this PR to help us see what you mean.

Reply:
Rewrite looks good.